### PR TITLE
Retry checking for locks during apt/dpkg operations in install.sh

### PR DESCRIPTION
**Pull request by UtheMan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,8 +12,9 @@ WORKDIR /opt/gpu
 COPY 10-nvidia-runtime.toml 10-nvidia-runtime.toml 
 COPY blacklist-nouveau.conf blacklist-nouveau.conf
 COPY fm_run_package_installer.sh fm_run_package_installer.sh
-COPY config.sh config.sh 
+COPY config.sh config.sh
```
Comment (alexeldeib) [inline Dockerfile:15]: ty

File: install.sh [hunk 2]
```diff
@@ -17,13 +19,21 @@ echo "Open gridd: $open_gridd"
...
+
+use_package_manager_with_retries wait_for_apt_locks install_linux_headers 10 3
 
 # install cached nvidia debian packages for container runtime compatibility
+install_cached_nvidia_packages() {
 for apt_package in $NVIDIA_PACKAGES; do
     dpkg -i --force-overwrite /opt/gpu/${apt_package}_${NVIDIA_CONTAINER_TOOLKIT_VER}*
 done
 dpkg -i --force-overwrite /opt/gpu/nvidia-container-runtime_${NVIDIA_CONTAINER_RUNTIME_VERSION}*
+}
```
Comment (alexeldeib) [inline install.sh:35]: can we drop a newline here

File: Dockerfile (hunk 1 above)
Comment (UtheMan) [reply Dockerfile:15]: wait what :D

**Review by alexeldeib — APPROVED** [review]
lgtm, we'll test this on the ab side once we integrate

**Pull request merged**